docs/api/environment-variables.md

```diff
@@ -48,20 +48,16 @@ Unsupported options are:
 
 ### `GOOGLE_API_KEY`
 
-Electron includes a hardcoded API key for making requests to Google's geocoding
-webservice. Because this API key is included in every version of Electron, it
-often exceeds its usage quota. To work around this, you can supply your own
-Google API key in the environment. Place the following code in your main process
+You can supply your own Google API key in the environment. Place the following code in your main process
```

Comment:
This loses the context that the previous revision had of "what is this key for."

Maybe something like

> You can provide an API key for making requests to Google's geocoding webservice. To do this, place the following code in your main process